### PR TITLE
Backend/images

### PR DESCRIPTION
**Pull request by Bonathan**
* Take pictures and send them
* Select a picture from the gallery and send it
* Check if the message is empty
* Rick-roll people with the /R command

### CODE REVIEW

File: app/src/main/assets/web/tremola.js [hunk 1]
```diff
@@ -186,30 +196,83 @@ function new_post(s) {
   closeOverlay();
 }
 
+function files() {
+    var recps = tremola.chats[curr_chat].members.join(' ')
```
Comment (LarsZauberer) [inline app/src/main/assets/web/tremola.js:220]: Need some comments

**Comment by LarsZauberer** [comment]
refactored files function

**Review by LarsZauberer — CHANGES_REQUESTED** [review]
Image size keep aspect ratio not implemented

**Comment by LarsZauberer** [comment]
Works fine

**Review by LarsZauberer — APPROVED** [review]
Great job

File: app/src/main/assets/web/tremola.js [hunk 2]
```diff
@@ -178,6 +178,16 @@ function menu_dump() {
 function new_post(s) {
   if (s.length == 0) { return; }
   var draft = unicodeStringToTypedArray(document.getElementById('draft').value); // escapeHTML(
+	var trimmed = '';
+  trimmed = draft.replace(/^\s+|\s+$/g, '');
```
Comment (LarsZauberer) [inline app/src/main/assets/web/tremola.js:194]: Put the button checking out of the `new_post` function

**Review by LarsZauberer — CHANGES_REQUESTED** [review]

**Review by carlostejera — CHANGES_REQUESTED** [review]
tremola.js function new_post: Block the button instead of filtering "empty" messages. The function should remain only functional.

**Review by LarsZauberer — APPROVED** [review]

**Pull request merged**